models/user.go

```diff
@@ -655,7 +655,7 @@ func NewGhostUser() *User {
 
 var (
 	reservedUsernames    = []string{"assets", "css", "explore", "img", "js", "less", "plugins", "debug", "raw", "install", "api", "avatars", "user", "org", "help", "stars", "issues", "pulls", "commits", "repo", "template", "admin", "error", "new", ".", ".."}
-	reservedUserPatterns = []string{"*.keys"}
+	reservedUserPatterns = []string{"*.keys", ".gpg"}
```

Comment:
Shouldn't this be `*.gpg`?

Reply:
Thanks for catching that @silverwind. Changing it now.